### PR TITLE
chore: run codemods for picasso icons

### PR DESCRIPTION
**Pull request by ozgurkececioglu**
[FX-2653]

### Description

I ran preset-safe codemod for icons folder

### How to test

- code review is enough

### Review

- [ ] Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/contribution/component-api.md)
- [ ] Annotate all `props` in component with documentation
- [ ] Create `examples` for component
- [ ] Ensure that deployed demo has expected results and good examples
- [ ] Ensure that tests pass by running `yarn test`
- [ ] Ensure that visuals tests pass by running `yarn test:visual`. If not - check the documentation [how to fix visual tests](https://github.com/toptal/picasso/blob/master/docs/contribution/visual-testing.md#fixing-broken-visual-tests-inside-a-pr)
- [ ] Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

[FX-2653]: https://toptal-core.atlassian.net/browse/FX-2653?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by augustobmoura — COMMENTED** [review]
It looks like the codemod added a semicolon in every import. This is not an error on eslint but is conflicting with our prettier rules. Maybe we should commit without then, wdyt?

This script removed then on my computer:
```bash
git show --stat ae0340792 | awk '/^ packages/ { print $1 }' | xargs sed -i 's/;//'
```

`ae0340792` is the commit reference hash, if you rebase it you might want to change it on the script

**Review by separatio — APPROVED** [review]
Ok. This PR was very straightforward. :shipit:

File: packages/picasso/src/Icon/styles.tsx [hunk 1]
```diff
@@ -1,4 +1,6 @@
-import { Theme, createStyles } from '@material-ui/core'
+import { Theme } from '@mui/material'
+
+import createStyles from '@mui/styles/createStyles'
```
Comment (TomasSlama) [inline packages/picasso/src/Icon/styles.tsx:3]:
```suggestion
import { Theme } from '@mui/material'
import createStyles from '@mui/styles/createStyles'
```

**Review by TomasSlama — APPROVED** [review]

File: packages/picasso/src/Icon/styles.tsx (hunk 1 above)
Comment (ozgurkececioglu) [reply packages/picasso/src/Icon/styles.tsx:3]: Wonderful catch! I fixed these changes by hand for many places but this one was missing. Thanks 🙏

**Comment by ozgurkececioglu** [comment]
> It looks like the codemod added a semicolon in every import. This is not an error on eslint but is conflicting with our prettier rules. Maybe we should commit without then, wdyt?
> 
> This script removed the semicolons on my computer:
> 
> ```shell
> git show --stat ae0340792 | awk '/^ packages/ { print $1 }' | xargs sed -i 's/;//'
> ```
> 
> `ae0340792` is the commit reference hash, if you rebase it you might want to change it on the script
> 
> We would want to run prettier after every codemod, but when I tried locally it changed other places that are not related to this commit, so maybe we should run prettier in all files on a different PR in the future
Thanks for the warning. I ran `prettier --write` by hand and committed the result for affected files to get rid of semicolons. Looks good now 👍

**Review by OndrejTuma — APPROVED** [review]
Nice 👍 this was quite easy to walk through 🙂

**Pull request merged**